.env

```diff
@@ -3,6 +3,6 @@ AWS_FUNCTION_NAME=PatronEligibilityService
 AWS_HANDLER=index.handler
 AWS_MEMORY_SIZE=512
 AWS_TIMEOUT=60
-AWS_RUNTIME=nodejs6.10
+AWS_RUNTIME=nodejs8.10
```

Comment:
Is there any change introduced in this app by updating node?

Reply:
Good question. Lambda has deprecated v6 support; Otherwise I wouldn't bother. The test suite succeeds and ad hoc testing suggests the app works the same. Among the changes from [v6 to v8](https://github.com/nodejs/wiki-archive/blob/master/Breaking-changes-between-v6-LTS-and-v8-LTS.md), I don't think we're relying on anything that has broken.